### PR TITLE
Component creation

### PR DESCRIPTION
**Pull request by sayanibhattacharjee**

### CODE REVIEW

File: modules/aws/elasticsearch_cloudwatch.hcl [hunk 1]
```diff
@@ -237,94 +250,3 @@ ingester aws_elasticsearch_cloudwatch module {
   }
 
 }
-
```
Comment (arjunmahishi) [inline modules/aws/elasticsearch_cloudwatch.hcl:240]: Why is this modules removed?

File: modules/aws/lambda_cloudwatch.hcl [hunk 2]
```diff
@@ -116,134 +122,3 @@ ingester aws_lambda_cloudwatch module {
     }
   }
 }
-
-ingester aws_lambda_resource_cloudwatch module {
```
Comment (arjunmahishi) [inline modules/aws/lambda_cloudwatch.hcl:120]: Same as above. I think some customers are using this.

File: modules/aws/alb_cloudwatch.hcl [hunk 3]
```diff
@@ -406,22 +428,26 @@ ingester aws_alb_internal_endpoint_cloudwatch module {
...
     source cloudwatch "throughput" {
       query {
         aggregator  = "Sum"
-        namespace   = "AWS/ApplicationELB"
+        namespace   = "AWS/ELB"
         metric_name = "RequestCount"
 
         dimensions = {
-          "LoadBalancer" = "$input{LoadBalancer}"
+          "LoadBalancerName" = "$input{LoadBalancerName}"
```
Comment (arjunmahishi) [inline modules/aws/alb_cloudwatch.hcl:441]: Does this change? (`namespace` and `dimensions`)

File: modules/aws/alb_cloudwatch.hcl [hunk 4]
```diff
@@ -406,22 +428,26 @@ ingester aws_alb_internal_endpoint_cloudwatch module {
     name = "$input{namespace}$input{endpoint}"
   }
 
-  gauge "throughput" {
-    unit = "tps"
+  gauge "throughput" "1" {
+    unit       = "rpm"
```
Comment (arjunmahishi) [inline modules/aws/alb_cloudwatch.hcl:432]: Does the unit change from "tps" to "rmp"

File: modules/aws/alb_cloudwatch.hcl [hunk 5]
```diff
@@ -301,11 +318,12 @@ ingester aws_alb_internal_cloudwatch module {
...
+    aggregator = "AVG"
     source cloudwatch "throughput" {
       query {
-        aggregator  = "Sum"
+        aggregator  = "Average"
```
Comment (arjunmahishi) [inline modules/aws/alb_cloudwatch.hcl:326]: Does the aggregation change? (question applies to sources below also)
`Sum` -> `Average`

**Review by arjunmahishi — COMMENTED** [review]
Need to double check `aggregator` and `unit` everywhere.

File: modules/aws/alb_cloudwatch.hcl (hunk 4 above)
Comment (sayanibhattacharjee) [reply modules/aws/alb_cloudwatch.hcl:432]: yes, confirmed from prod

File: modules/aws/alb_cloudwatch.hcl (hunk 5 above)
Comment (sayanibhattacharjee) [reply modules/aws/alb_cloudwatch.hcl:326]: yes, confirmed from prod

**Pull request closed without merging**